pallets/dkg-metadata/src/mock.rs

```diff
@@ -123,6 +124,17 @@ where
 	}
 }
 
+parameter_types! {
+	#[derive(Clone, Encode, Decode, Debug, Eq, PartialEq, scale_info::TypeInfo, Ord, PartialOrd, MaxEncodedLen, Default)]
+	pub const MaxKeyLength : u32 = 10_000;
+	#[derive(Clone, Encode, Decode, Debug, Eq, PartialEq, scale_info::TypeInfo, Ord, PartialOrd, MaxEncodedLen, Default)]
+	pub const MaxSignatureLength : u32 = 100;
+	#[derive(Clone, Encode, Decode, Debug, Eq, PartialEq, scale_info::TypeInfo, Ord, PartialOrd, MaxEncodedLen, Default)]
+	pub const MaxAuthorities : u32 = 1000;
+	#[derive(Clone, Encode, Decode, Debug, Eq, PartialEq, scale_info::TypeInfo, Ord, PartialOrd, MaxEncodedLen, Default)]
+	pub const MaxReporters : u32 = 1000;
+}
+
```

Comment:
Same goes here for these sizes.

Reply:
You can actually use `MaxSignatureLength::get()` from above, to unify these in one place.